packages.json

```diff
@@ -104,5 +104,11 @@
 	"698780": {
 		"game_name":    "Doki Doki Literature Club",
 		"command":      "./DDLC.sh"
+	},
+	"22320": {
+        "game_name": "The Elder Scrolls III: Morrowind",
```

Comment:
I know it's hard to spot, but in this specific file I use tabs for indentation, and spaces to line up values in json key: value pairs.